sdk/keyvault/service.projects

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <!-- Remove mgmt plane tests from nightly live runs: https://github.com/Azure/azure-sdk-for-net/issues/12211 -->
+    <ProjectReference Remove="$(MSBuildThisFileDirectory)Azure.Management.*\**\*.csproj" Condition="'$(AZURE_KEYVAULT_TEST_MODE)' == 'Live'" />
```

Comment:
Actually can we scope this to just the test projects?

Reply:
I could, but there is only the one (and would likely only ever be one). Care enough for me to push a change now that test runs (playback and live) have started?